### ISSUE TITLE
frontend: forgotten password 

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#38]
## User story
1. As a {HR-employee}
2. I want / need {to have the possibility to get access to DIdentity if I forgot my current password}
3. So that {I can still do my work}

## Acceptance criteria
* button on login page called "Password forgotten?" links to a page where HR-employee can enter his/her email-address to which the new password is sent
* if no account with the given email-address exists, error is displayed. New email-address can be typed in again
* if successfull, a message is displayed: "link to create new password has been sent to your email-address"
* user reaches a webpage via a link in the email (sent by the backend) where he/she types in new password twice and confirms. This is the same page as the one used for the first initialization of the account. Green notification "success" in case new password has been successfully set.

 
## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
